spec/models/user_spec.rb

```diff
@@ -1,5 +1,46 @@
 require "rails_helper"
+require "support/factory_girl"
 
 RSpec.describe User, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+  it 'is valid' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.